wolfcrypt/src/asn.c

```diff
@@ -24911,8 +24936,21 @@ static int EncodeCert(Cert* cert, DerCert* der, RsaKey* rsaKey, ecc_key* eccKey,
     /* set the extensions */
     der->extensionsSz = 0;
 
+    /* RFC 5280 : 4.2.1.9. Basic Constraints
```

Comment:
These new sections using `keyUsage` need wrapped with `#ifdef WOLFSSL_CERT_EXT`.

```
ERRORS
--
Total : 5 error(s)
13 [1;38;5;243m--enable-certgen --enable-certreq --enable-sha512 --enable-opensslextra --enable-hashdrbg --enable-sessioncerts --enable-keygen --enable-crl --enable-ecc --enable-dsa --enable-hkdf --disable-fpecc --enable-eccencrypt --enable-savecert --enable-supportedcurves --disable-fastmath[0m
1 wolfcrypt/src/asn.c:25050:15: error: ‘Cert {aka struct Cert}’ has no member named ‘keyUsage’
2 wolfcrypt/src/asn.c:25050:59: error: ‘Cert {aka struct Cert}’ has no member named ‘keyUsage’
3 wolfcrypt/src/asn.c:26234:15: error: ‘Cert {aka struct Cert}’ has no member named ‘keyUsage’
4 wolfcrypt/src/asn.c:26234:59: error: ‘Cert {aka struct Cert}’ has no member named ‘keyUsage’
Total : 0 warning(s)
```

Note line numbers do not match since the test is using master + this PR.